src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,112 @@
+package com.google.codeu.servlets;
+
+
+import java.io.IOException;
+
+
+import javax.servlet.annotation.WebServlet;
+
+import javax.servlet.http.HttpServlet;
+
+import javax.servlet.http.HttpServletRequest;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+
+import com.google.appengine.api.users.UserServiceFactory;
+
+import com.google.codeu.data.Datastore;
+
+import com.google.codeu.data.User;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
```

Comment:
Import statements should be lexicographically sorted. If you use linter, it can sort these imports for you. Both Intellija nd Visual Studio have Java Lint plugins, not sure about Eclipse.